src/tutorial/cli-args.md

```diff
@@ -51,23 +50,23 @@ $ cargo run -- some-pattern some-file
 pattern: "some-pattern", path: "some-file"
 ```
 
-## CLI arguments as data type
+## CLI arguments as data types
 
 Instead of thinking about them as a bunch of text,
 it often pays off to think of CLI arguments as a custom data type
 that represents the inputs to your program.
 
-Look at `grrs foobar test.txt`:
-There are two arguments,
-first the `pattern` (the string to look for),
-and then the `path` (the file to look in).
+Look at `grrs foobar test.txt`.
+There are two arguments:
```

Comment:
This keep the sentence as an imperative which is not a friendly way of writing.  Maybe
```suggestion
Looking at `grrs foobar test.txt`,
there are two arguments:
```